tools/data_converter/sunrgbd_data_utils.py

```diff
@@ -191,9 +191,9 @@ def process_single_scene(sample_idx):
                     ],
                                                              axis=0)
                     annotations['dimensions'] = 2 * np.array([
-                        [obj.l, obj.h, obj.w] for obj in obj_list
+                        [obj.l, obj.w, obj.h] for obj in obj_list
```

Comment:
If I understand correctly, this change will cause BC-breaking in the processed SUN RGB-D data? Why this PR which targets on docs will need to change this?

Reply:
No, this 'dimensions' is unused. I changed this because this is very confusing.

Reply:
When I added comments in the code fraction in the doc, I wrote ... boxes (in depth coordinate system)... However this 'dimensions' is in camera coordinate system

Reply:
ICIC